### ISSUE TITLE
"pretty" option for serialize()

### ISSUE THREAD

**Issue by jiseonmin** [MesserLab/SLiM#429]
Hi Ben,

I am suggesting adding a pretty option to `serialize()` based on our discussion in https://github.com/kr-colab/spatial_sims_standard/pull/7

Something like:
```
catn("-------------\n");
	for (k in defaults.allKeys) {
		catn("  " + k + ": " + defaults.getValue(k));
	}
catn("-------------\n");
```
for a dictionary called `defaults`

**Comment by bhaller** [comment]
Will do, will be in SLiM 4.2.  :->

**Comment by bhaller** [comment]
OK, this has been pushed.  You can now do:

```
catn("-------------\n");
catn(defaults.serialize("pretty"));
catn("-------------\n");
```

to get something similar (although not identical) to what you were printing before.  Let me know if there are any problems with this; until it is released in SLiM 4.2, I can make changes to it as needed.  :->
